### DISCUSSION TITLE
Sliding Window / Hop Configuration for Yamnet

### DISCUSSION THREAD

**Discussion by technosaby** [technosaby/gsoc2022#18] [General]
Tensorflow frames the waveform into sliding windows of length 0.96 seconds and hop 0.48 seconds, and then run the core of the model on a batch of these frames. It is an important thing to check if this can be configured and how ?

**Comment by brucearctor** [comment]
https://groups.google.com/g/audioset-users/c/pRDX6AkaM1s 

"PATCH_WINDOW_SECONDS is a property of the classifier, it can't be changed without training a new model with a differently-sized input layer.  If you use a value other than 0.96 in the example code, it will likely just mess up the prediction of the number of frames and their correct alignment, it won't change the classifier scores.
"

and I assume that comment is from this guy: https://research.google/people/DanEllis/ ... 

From this group --> https://groups.google.com/g/audioset-users ( which might interest you )
